sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/converter/ast/nodes/operator/logic/GremlinSqlBinaryOperator.java

```diff
@@ -176,6 +198,23 @@ void handleEmbeddedGremlinSqlBasicCall(GremlinSqlBasicCall gremlinSqlBasicCall,
         return graphTraversals;
     }
 
+    public String getOperandName(final GremlinSqlNode operand) throws SQLException {
+        if (operand instanceof GremlinSqlIdentifier) {
+            final GremlinSqlIdentifier gremlinSqlIdentifier = (GremlinSqlIdentifier) operand;
+            return gremlinSqlIdentifier.isStar() ? "*" : gremlinSqlIdentifier.getColumn();
+        } else if (operand instanceof GremlinSqlLiteral) {
+            return ((GremlinSqlLiteral) operand).getValue().toString();
+        } else if (operand instanceof GremlinSqlBasicCall) {
+            return ((GremlinSqlBasicCall) operand).getRename();
+        }
+        throw new SQLException("Error, unsupported operand type, cannot rename column.");
```

Comment:
Should we refine error, including what specific type is not supported? Would be good to include guidance on whether it is an error or not supported feature.

Reply:
Also add to bundle.